docs/release/support_matrix.md

```diff
@@ -1,7 +1,428 @@
+(the RPM distribution of DAOS includes libfabric RPM packages with the correct version):
+
+* [libfabric 1.18.0](https://github.com/ofiwg/libfabric/releases/tag/v1.18.0) (Apr 08, 2023)
+* [libfabric 1.18.1rc1](https://github.com/ofiwg/libfabric/releases/tag/v1.18.1rc1) (release candidate)
+
+Not all libfabric core providers listed in
+[fi\_provider(7)](https://ofiwg.github.io/libfabric/main/man/fi_provider.7.html)
+are supported by DAOS. The following providers are supported:
+
+* `ofi+tcp` is supported on all networking hardware.
+  It does not use RDMA, so on an RDMA-capable network this provider typically
+  does not achieve the maximum performance of the fabric.
+* `ofi+verbs` is supported for RDMA communication over InfiniBand fabrics.
+  Note that as an alternative to libfabric, the UCX networking stack can be
+  used on InfiniBand fabrics as described in the next subsection.
+* The `ofi+cxi` provider is supported for RDMA communication over Slingshot.
+* The `ofi+opx` (Omni-Path Express) provider is enabled as a _Technology Preview_
+  for RDMA transport over Omni-Path fabrics, for testing and evaluation purposes.
+  In production environments using Omni-Path networking, please continue to use
+  the `ofi+tcp` provider until the `ofi+opx` provider is fully supported.
+
+!!! note
+    The `ofi+psm2` provider for Omni-Path fabrics has known issues
+    when used in a DAOS context, and is not supported by DAOS Version 2.4.
+
+!!! note
+    The `ofi+psm3` provider for Ethernet fabrics has not been validated with
+    and is not supported by DAOS Version 2.4.
+
+
+### UCF Unified Communication X (UCX)
+
+For InfiniBand fabrics, DAOS 2.4 also supports [UCX](https://openucx.org/),
+which is maintained by the Unified Communication Framework (UCF) consortium.
+
+DAOS Version 2.4 has been validated primarily with UCX Version 1.14.0-1,
+which is included in the MLNX\_OFED 5.8 levels listed in the next section.
+UCX Version 1.15.0-1 (included in MLNX\_OFED 5.9) has not been validated with DAOS 2.4 yet.
+
+* The `ucx+dc_x` provider has been validated and is supported with DAOS Version 2.4.
+  It is the recommended fabric provider on InfiniBand fabrics.
+
+* The `ucx+tcp` provider can be used for evaluation and testing purposes,
+  but it has not been fully validated with DAOS Version 2.4 and is not supported
+  for use in production environments.
+
+
+### NVIDIA/Mellanox OFED (MLNX\_OFED)
+
+On [NVIDIA/Mellanox InfiniBand](https://www.nvidia.com/en-us/networking/products/infiniband/)
+fabrics, DAOS requires that the
+[Mellanox OFED (MLNX\_OFED)](https://www.mellanox.com/products/infiniband-drivers/linux/mlnx_ofed)
+software stack is installed on the DAOS servers and DAOS clients.
+
+DAOS Version 2.4 has been validated with MLNX\_OFED Version 5.8-1,
+and  both 5.8-1.0.1.1 and 5.8-1.1.2.1 are supported.
+Versions older than 5.8-1 are not supported by DAOS 2.4.
+MLNX\_OFED 5.8-2 and 5.9 has not been validated with DAOS 2.4 yet.
+
```

Comment:
we should also add information MOFED 23.x.x.x (the non-LTS version). I think it works, but idk whether it was officially validated.

Reply:
done